src/jail_run.c

```diff
@@ -182,7 +182,21 @@ jail_run(
 
     /* Files and directories */
     for (path_ptr = rules->paths; *path_ptr; path_ptr++) {
+        static const char home_prefix[] = "${HOME}/";
         const JailPath* jp = *path_ptr;
+        /* FIXME: Whitelisting can be about directories or files.
+         *        This works in the expected manner only as long as
+         *        sailjail features are used only for whitelisting
+         *        directories. See jail_rules_parse_section().
+         */
+        if (jp->allow && jp->require) {
+            if (!strncmp(jp->path, home_prefix, sizeof home_prefix - 1)) {
+                char* opt = g_strconcat("--mkdir=", jp->path, NULL);
```

Comment:
If that relies on a firejail patch, .spec needs to require it

Reply:
ack, once the firjail pr gets merged we have a version to use

Reply:
Will you @spiiroin bump up the firejail requirement.

Reply:
Will update requirement.